src/DynamoCore/Library/LibraryCustomization.cs

```diff
@@ -87,6 +87,14 @@ public static bool ResolveForAssembly(string assemblyLocation, ref string custom
                 var fn = Path.GetFileNameWithoutExtension(assemblyLocation);
                 resourceAssemblyPath = fn + Configurations.IconResourcesDLL;
 
+                if (DynamoPathManager.Instance.ResolveLibraryPath(ref resourceAssemblyPath))
+                    return true;
+
+                // If customization dll wasn't found among common paths,
+                // try to search it at the same path as dll's.
+                // This case is made just for 3 party dll.
+                var dirName = Path.GetDirectoryName(assemblyLocation);
+                resourceAssemblyPath = Path.Combine(dirName, fn + Configurations.IconResourcesDLL);
```

Comment:
If the first `ResolveLibraryPath` failed, then the second call will result in the same failure. So the second call is not needed. Can you try this:

```csharp
// First try side-by-side search for customization dll.
var dirName = Path.GetDirectoryName(assemblyLocation);
resourceAssemblyPath = Path.Combine(dirName, fn + Configurations.IconResourcesDLL);
if (File.Exists(resourceAssemblyPath))
    return true;

// Side-by-side customization dll not found, try other resolution paths.
return DynamoPathManager.Instance.ResolveLibraryPath(ref resourceAssemblyPath);
```